lib/index.js

```diff
@@ -7,8 +7,17 @@ const _ = require('lodash');
 const DefaultModel = require('./default-model');
 const defaults = require('./defaults');
 
-const getFields = key => ({
+const conditionalTranslate = (key, t) => {
```

Comment:
yes there should be - i18nFuture will return the key if it fails, sometimes we want nothing, but I'm not fixing 18n